### ISSUE TITLE
Feature request: Start the recording instantly

### ISSUE THREAD

**Issue by sirpooya** [wulkano/Kap#1144]
Every time that click the record button it takes 3 seconds for Kap's UI overlay to go off and for recording to start.
It would be amazing if the recording could start the second I click on the record button.

**Comment by dinbtechit** [comment]
Yeah, it's a bit annoying it takes 3 seconds to start after clicking the button. Is this a limitation of electron?
